server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -466,6 +507,16 @@ def formula(
         #   [my age] + 1 / 2
         # into
         #   CAST("my age" AS float) + 1 / 2
+
+        # We add quotes to identify columns
+        # for c in table.columns:
+        #     step.formula = (
+        #         f" {step.formula} ".replace(f" {c} ", f" \"{c}\" ")
+        #         .replace(f"({c} ", f"(\"{c}\" ")
+        #         .replace(f" {c})", f" \"{c}\")")
+        #     )
+        #     step.formula = " ".join(step.formula.split())
```

Comment:
To me, this does not sound like something we want to do. Escaping column names should be up to PyPika, especially since the quoting character to use depends on the SQL dialect

Reply:
Yeah, exactly, that's why it's commented, i removed these lines (with comments) here : https://github.com/ToucanToco/weaverbird/pull/1390/commits/fda3338f1dc14a8243e54ffbd76caf3769329d11